README.md

```diff
@@ -586,6 +587,45 @@ crud.len('customers')
 ...
 ```
 
+### Count
+
+`CRUD` supports multi-conditional count, treating a cluster as a single space.
+The same as with `select()` the conditions may include field names or numbers,
+as well as index names. The recommended first condition is a TREE index; this
+helps to reduce the number of tuples to scan. Otherwise a full scan is performed.
+If compared with `len()`, `count()` method scans the entire space to count the
+tuples according user conditions. This method does yield that's why result may
+be approximate. Number of tuples before next `yield()` is under control with
+option `count_to_yield`.
+
+```lua
+local result, err = crud.count(space_name, conditions, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space
+* `conditions` (`?table`) - array of [conditions](#select-conditions)
+* `opts`:
+  * `count_to_yield` (`?number`) - number of tuples processed to yield after,
+    `count_to_yield` should be > 0, default value is 100
```

Comment:
Sorry for the late response, but I would like to name it `yield_every`.

Reply:
(And don't forget to update commit message too)

Reply:
Updated.